packages/crud/src/crud/reflection.helper.ts

```diff
@@ -22,6 +15,14 @@ import {
 } from '../constants';
 import { CrudActions } from '../enums';
 
+const {
+  CUSTOM_ROUTE_AGRS_METADATA = CONSTANTS['CUSTOM_ROUTE_ARGS_METADATA'],
```

Comment:
maybe can change to 
CUSTOM_ROUTE_AGRS_METADATA = CONSTANTS['CUSTOM_ROUTE_ARGS_METADATA'] || CONSTANTS['CUSTOM_ROUTE_AGRS_METADATA']

Reply:
It's destructuring CONSTANTS and if available it takes CUSTOM_ROUTE_AGRS_METADATA, if not it takes the default value which is CONSTANTS['CUSTOM_ROUTE_ARGS_METADATA']

Here a small example of the first case https://www.typescriptlang.org/play?#code/MYewdgzgLgBAwgeQHIGUAqBBJaUwLwwDeAUDPAKroICyA+gEoLloCitGA4vSrdS5gBEMmAFwwA5FACm0ccQC+AbmLFQkWIRikKVOo2ZtO3XvwxDM+eMnRYcAbXFxKaGgyat29Djz6DhGcQBdABoFS0RUTGwUZVVwCBAAGykAOkSQAHMACiddNwN2Lh9TcwwASkUgA
and here of the second https://www.typescriptlang.org/play?#code/MYewdgzgLgBAwgeQHIGUAqBBJaUwLwwDeAUDPAKroICyA+gEoLloCitG9A4irdS5gBEMmAFwwA5FACm0ccQC+AbmLFQkWIRikKVOo2ZsMnejz6DhGfPGTosOANri4lNDQZNW7Lqf4YhmcQBdABoFK0RUTGwUZVVwCBAAGykAOkSQAHMACmdddwN2Yx9zTABKRSA

As you can see in both cases CUSTOM_ROUTE_AGRS_METADATA is defined.

Reply:
> It's destructuring CONSTANTS and if available it takes CUSTOM_ROUTE_AGRS_METADATA, if not it takes the default value which is CONSTANTS['CUSTOM_ROUTE_ARGS_METADATA']
> 
> Here a small example of the first case https://www.typescriptlang.org/play?#code/MYewdgzgLgBAwgeQHIGUAqBBJaUwLwwDeAUDPAKroICyA+gEoLloCitGA4vSrdS5gBEMmAFwwA5FACm0ccQC+AbmLFQkWIRikKVOo2ZtO3XvwxDM+eMnRYcAbXFxKaGgyat29Djz6DhGcQBdABoFS0RUTGwUZVVwCBAAGykAOkSQAHMACiddNwN2Lh9TcwwASkUgA and here of the second https://www.typescriptlang.org/play?#code/MYewdgzgLgBAwgeQHIGUAqBBJaUwLwwDeAUDPAKroICyA+gEoLloCitG9A4irdS5gBEMmAFwwA5FACm0ccQC+AbmLFQkWIRikKVOo2ZsMnejz6DhGfPGTosOANri4lNDQZNW7Lqf4YhmcQBdABoFK0RUTGwUZVVwCBAAGykAOkSQAHMACmdddwN2Yx9zTABKRSA
> 
> As you can see in both cases CUSTOM_ROUTE_AGRS_METADATA is defined.

thanks @ckaeslin , you are right